R/cluster.R

```diff
@@ -0,0 +1,30 @@
+#' cluster based on vectors
+#' @export
+cluster <- function(..., type = "kmeans", n_cluster = 3) {
+  data <- list(...) %>% as.data.frame() %>% as.matrix()
+  na_removed <- data %>% na.omit()
+  if(nrow(na_removed) == 0) {
```

Comment:
Does k-means work for example when nrow() is 1 or below n_cluster?
If not, we can limit this condition further.